lib/osut/utils.rb

```diff
@@ -859,14 +1241,24 @@ def defaultConstructionSet(s = nil)
     type     = s.surfaceType
     ground   = false
     exterior = false
+    adjacent = s.adjacentSurface.empty? ? nil : s.adjacentSurface.get
+    aspace   = adjacent.nil? || adjacent.space.empty? ? nil : adjacent.space.get
+    typ      = adjacent.nil? ? nil : adjacent.surfaceType
```

Comment:
Not checking if a surface inherits a construction from its adjacent:

- surface
- space
- spacetype
- story

... is a big no-no, creating a ton of headaches for both TBD and BTAP. Initial tests are green.